### PR TITLE
Recurring ampm

### PR DESCRIPTION
**Pull request by leeacto**
Did not add spec for JS fix, as I didn't see any JS tests.

The change to AM/PM over 2400 broke the recurring template. This fixes it.

### CODE REVIEW

File: app/assets/javascripts/availabilities/availability_recurrence.js [hunk 1]
```diff
@@ -23,22 +23,40 @@ AvailabilityRecurrence.prototype.toggleRecurrencePattern = function() {
...
+  return pad(hours) + ":" + pad(minutes) + " " + ampm;
+}
+
 AvailabilityRecurrence.prototype.availabilityTime = function() {
   var date = $("#availability_start_time_1s").val();
-  var hour = $("#availability_start_time_4i option:selected").val();
-  var minute = $("#availability_start_time_5i option:selected").val();
-
-  return new Date(date + " " + hour + ":" + minute);
+  var hour = $("#availability_start_time_4i_ option:selected").val();
```
Comment (redsquirrel) [inline app/assets/javascripts/availabilities/availability_recurrence.js]: Here's where I expose my ignorance of modern Rails form helpers: What's with these trailing underscore elements? 4i_, 5i_, 6i_? Can you explain why you're using ..._start_time_1s and then ..._start_time_4i_?

Comment (ltw) [inline app/assets/javascripts/availabilities/availability_recurrence.js]: Those are the namespaces that Rails forms necessarily gives each `select` in the "datetime" input.

Comment (leeacto) [inline app/assets/javascripts/availabilities/availability_recurrence.js]: It's a helper made by @echappen. 
https://github.com/Devbootcamp/mentor-pairing/blob/master/app/helpers/availabilities_helper.rb

start_time_1s to 3s likely represents Month/Day/Year. 4/5/6 is Hour/Minute/AMPM.

So not not ignorance, but something specific to the app :)

image

Comment (ltw) [inline app/assets/javascripts/availabilities/availability_recurrence.js]: It was named `4i`-`6i` to fit with the rails helper naming scheme.

**Pull request merged**